schemas/mw-revision-visibilityset.yaml

```diff
@@ -0,0 +1,62 @@
+title: MediaWiki Revision Visibility Set
```

Comment:
Maybe this is already a MW name, but if it isn't 'Set' here could be confusing.  To me, this title indicates that this record is for a set of 'revision visibilities' (whatever that might be).  Does MW call this a 'set event'?  If so, ignore me.  If not, maybe 'MediaWiki Revision Visibility Change'?